www/modules/custom/borg_tugboat/templates/tugboat-preview-ready-page.tpl.php

```diff
@@ -27,16 +31,14 @@
 
   <div class="borg-demo-done-left">
     <p>
-      This demo sandbox will act as a new Backdrop installation would. You will
-      need to run the installer and set an administrator account as part of the
-      setup. Note that sending email from demo sandboxis not allowed, so
-      <strong>password recovery will not work</strong>. Be sure to remember your
-      password.
+      Note that sending emails from demo sandboxes is not allowed, so
```

Comment:
Should we pass this through `t()`, in case we want to translate in the future?

Reply:
Yes, but I've created a [separate issue](https://github.com/backdrop-ops/backdropcms.org/issues/664) for this since it's a larger problem than just the text changed here.